chain/actors/builtin/builtin.go

```diff
@@ -0,0 +1,23 @@
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/filecoin-project/go-state-types/network"
+)
+
+type Version int
+
+const (
+	Version0 = iota
+)
+
+// Converts a network version into a specs-actors version.
+func VersionForNetwork(version network.Version) Version {
+	switch version {
+	case network.Version0, network.Version1:
```

Comment:
Yes. Network version != actor version. Also, actor version is going to go from 0 (well, 0.9) to 2. That's going to be rather strange..

Reply:
If it will just be used as a unique key, maybe use a string "v2" instead?

Reply:
We're now encoding this inside the state tree itself.

@magik6k what version do we actually _want_ in the state tree. We could use:

* Network version (seems like overkill).
* Actors version (what we're using here).
* A specific "state tree" version (for the HAMT). That seems like the _best_ approach, honestly.

Thoughts?

Reply:
A specific "state tree" version sounds like what we want here